src/main/scala/viper/gobra/ast/internal/PrettyPrinter.scala

```diff
@@ -250,7 +250,7 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
     case Seqn(stmts) => ssep(stmts map showStmt, line)
     case Label(label) => showProxy(label)
     case If(cond, thn, els) => "if" <> parens(showExpr(cond)) <+> block(showStmt(thn)) <+> "else" <+> block(showStmt(els))
-    case While(cond, invs, measure, body) => "while" <> parens(showExpr(cond)) <> line <>
+    case While(cond, invs, measure, body, label) => opt(label)(text) <> opt(label.map(_ => ": "))(text) <> "while" <> parens(showExpr(cond)) <> line <>
```

Comment:
couldn't this be simplify as follows?
```suggestion
    case While(cond, invs, measure, body, label) => opt(label.map(l => s"${l}: "))(text) <> "while" <> parens(showExpr(cond)) <> line <>
```